src/ast/irbuilderbpf.h

```diff
@@ -89,6 +89,18 @@ class IRBuilderBPF : public IRBuilder<> {
                              Value *key,
                              SizedType &type,
                              const location &loc);
+
+  Value *CreatePerCpuMapLookupElem(Value *ctx,
+                                   const std::string &map_name,
+                                   Value *key,
+                                   Value *cpu,
+                                   SizedType &type,
+                                   const location &loc);
```

Comment:
Is this unused? I deleted it from my local checkout and things still compile && work

Reply:
Good catch. I think this was leftover from a previous revision